packaging/torchvision/meta.yaml

```diff
@@ -8,6 +8,7 @@ source:
 requirements:
   build:
     - {{ compiler('c') }} # [win]
+    - libpng
```

Comment:
Are you sure this is only a build-time dependency? Have you copied the libs for the conda package?